### PR TITLE
openedge: upgrade to 2.19.1

### PR DESCRIPTION
**Pull request by gquerret**

### CODE REVIEW

File: openedge.properties [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-publicVersions=2.14.4
+archivedVersions=2.14.4
+publicVersions=2.19.1
 
 defaults.mavenGroupId=eu.rssw.sonar.openedge
 defaults.mavenArtifactId=sonar-openedge-plugin
 
 2.14.4.description=New expression engine in ABL Parser
-2.14.4.sqVersions=[8.9,LATEST]
+2.14.4.sqVersions=[8.9,9.6]
```
Comment (ganncamp) [inline openedge.properties:11]: The current version is 9.8. Ending this range with 9.6 means users on 9.7 and 9.8 won't be prompted to upgrade. Is that really what you intend?

**Review by ganncamp — CHANGES_REQUESTED** [review]
I need a link to your project on SonarCloud

**Comment by gquerret** [comment]
> I need a link to your project on SonarCloud

https://sonarcloud.io/summary/new_code?id=eu.rssw%3Asonar-openedge
And I see that the latest quality gate is failing. Version 2.19.0 was ready to release, then I saw that some unit tests were not included (that was breaking the quality gate as the code coverage wasn't good enough). Those missing unit tests were added, so now I'm not sure I can really do something about the failing quality gate (except creating a fake release in order to have a green light).

File: openedge.properties (hunk 1 above)
Comment (gquerret) [reply openedge.properties:11]: My understanding was that 2.14.4 (the previous version) would be proposed up to 9.6 (where it has been tested). Only the latest version would be proposed on SQ 9.7+. Am I wrong ?

Comment (ganncamp) [reply openedge.properties:11]: This previous compatibility ends with `LATEST`, which means it's being offered to folks on 9.8. So if you now end compatibility at 9.6, that "strands" your users on 9.7 & 9.8

**Review by ganncamp — CHANGES_REQUESTED** [review]
Regarding your QG, could you do a 2.19.2 to get the QG back to green?

**Comment by gquerret** [comment]
Released 2.19.2, quality gate is green.
Also fixed the sqVersions property, and force-pushed.

File: openedge.properties [hunk 2]
```diff
@@ -12,3 +12,9 @@ defaults.mavenArtifactId=sonar-openedge-plugin
 2.14.4.date=2021-11-23
```
Comment (ganncamp) [inline openedge.properties:12]: Only one version can be compatible with `LATEST` at a time. You need to make this range end with 9.8

File: openedge.properties [hunk 3]
```diff
@@ -12,3 +12,9 @@ defaults.mavenArtifactId=sonar-openedge-plugin
...
 2.14.4.changelogUrl=https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.14.4
 2.14.4.downloadUrl=https://github.com/Riverside-Software/sonar-openedge/releases/download/V2.14.4/sonar-openedge-plugin-2.14.4.jar
+
+2.19.1.description=Version 2.19.1
```
Comment (ganncamp) [inline openedge.properties:16]: You've released 2.19.2, but nothing in this block has been updated to point to it...?

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: openedge.properties (hunk 2 above)
Comment (gquerret) [reply openedge.properties:12]: Done

File: openedge.properties (hunk 3 above)
Comment (gquerret) [reply openedge.properties:16]: Just changed that.

Comment (gquerret) [reply openedge.properties:16]: I'll rename the version 2.19.2, that will be better...

Comment (gquerret) [reply openedge.properties]: Done

**Review by ganncamp — APPROVED** [review]
Woo hoo!

**Pull request merged**